src/util.jl

```diff
@@ -150,20 +146,29 @@ condition `a[I,i]=A[i]` is applied to the vector component _normal_ to the domai
 boundary. For example `aₓ(x)=Aₓ ∀ x ∈ minmax(X)`. A zero Neumann condition
 is applied to the tangential components.
 """
-function BC!(a,A)
+function BC!(a,A,saveexit=false)
     N,n = size_u(a)
     for j ∈ 1:n, i ∈ 1:n
         if i==j # Normal direction, Dirichlet
-            for s ∈ (1,2,N[j])
+            for s ∈ (1,2)
                 @loop a[I,i] = A[i] over I ∈ slice(N,s,j)
             end
+            (!saveexit || i>1) && (@loop a[I,i] = A[i] over I ∈ slice(N,N[j],j)) # overwrite exit
         else    # Tangential directions, Neumann
             @loop a[I,i] = a[I+δ(j,I),i] over I ∈ slice(N,1,j)
             @loop a[I,i] = a[I-δ(j,I),i] over I ∈ slice(N,N[j],j)
         end
     end
 end
 
+function exitBC!(u,u⁰,U,Δt)
+    N,_ = size_u(u)
+    exit = slice(N.-1,N[1],1,2)               # exit slice excluding ghosts
```

Comment:
`exit` is a Julia keyword, would be safer not to override it even if harmless here :)

Reply:
Good point!

Reply:
replace by `outflow` or `outlet` maybe?